ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java

```diff
@@ -78,11 +88,18 @@ public PreparedStatement<?> prepare(
 
       return PreparedStatement.of(stmt.getUnMaskedStatementText(), root);
     } catch (final ParseFailedException e) {
-      if (!e.getSqlStatement().isEmpty()) {
-        throw e;
-      }
+      //      if (!e.getSqlStatement().isEmpty()) {
+      //        throw e;
+      //      }
+      throw new ParseFailedException(
+          e.getUnloggedMessage(), stmt.getMaskedStatementText(), e.getCause());
```

Comment:
Needed the remove the "re-throw" above and pass in `getUnloggedMessage()` here to make tests pass -- pervious PRs don't have this change, and I am not sure why it's required for 6.0. Thoughts?

Reply:
Hmm that is likely because the tests are looking for the full message which we arn't supposed to give anymore. I think we need to give the message in the message slot and the unloggged in the unlogged slot and fix the tests.

Reply:
Dug into it -- think I figured it out -- cf other comments.